common/lib/xmodule/xmodule/js/src/sequence/edit.js

```diff
@@ -0,0 +1,21 @@
+/* globals XModule, Reflect */
+
+/* Note: this was originally a one-line Coffeescript file, which we opted not to transpile directly
+ * to JS because of the grossness of the Coffeescript inheritance boilerplate. Instead I've tried to
+ * approximate the class/extends model the Coffeescript had in ES5 using reflection.
+ */
+
+(function() {
+    'use strict';
+
+    this.SequenceDescriptor = (function() {
+        function SequenceDescriptor() {
+            return Reflect.construct(XModule.Descriptor, arguments, SequenceDescriptor);
+        }
+
+        Reflect.setPrototypeOf(SequenceDescriptor.prototype, XModule.Descriptor.prototype);
```

Comment:
Have to admit I don't understand this that well, I patched this together from [a thread on ESDiscuss](https://esdiscuss.org/topic/extending-an-es6-class-using-es5-syntax) and I'm just now realizing while digging into the MDN docs for `Reflect` that this won't work in IE or Safari 9.

Gonna have to take another go at this, I think.